packages/f-checkout/src/components/Checkout.vue

```diff
@@ -43,21 +43,21 @@
 
                 <user-note data-test-id="user-note" />
 
-                <button-component
+                <f-button
```

Comment:
Can we name this `button-component` and import `import ButtonComponent from '@justeat/f-button';`?

Think it reads a little better then in the template (and we don't use `f-` generally when importing components elsewhere).

Reply:
FButton/f-button looked to match one of [the recommended naming schemes](https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended) and I changed it as the result of discussion in https://github.com/justeat/fozzie-components/pull/507

Reply:
I will merge this for now and can change later if the team thinks `...-component` is better to use